### DISCUSSION TITLE
Create Scheduler for updating table

### DISCUSSION THREAD

**Discussion by Qura-tul-ain** [supabase/supabase#13342] [Questions]
Hi, 

i have a Job table in which i have a column named "days" .... what i want is to decrement days of every post by one after 24 hours.

select cron.schedule (
    'daily-cleanup', -- name of the cron job
    '0 23 * * *', -- daily
    $$ update from jobs set remainingDays = remainingDays - 1 $$
);

this is my code for updating value... but this is not working. can anyone help me with this issue ?

Thanks in advance.

**Comment by GaryAustin1** [comment]
Have you checked your update SQL on its own?   You are using capital letters which is not recommended in Postgres.  To get names to work with them you need to use double quotes.  “remainingDays”. 

You might need a ; after -1 also.

**Comment by Qura-tul-ain** [comment]
Thanks for your response. i have updated name to  “remainingDays” plus added  ; but it's still not working.
